mmaction/datasets/pipelines/loading.py

```diff
@@ -270,6 +271,239 @@ def _get_test_clips(self, num_frames):
         return clip_offsets
 
 
+@PIPELINES.register_module()
+class SampleProposalFrames(SampleFrames):
+    """Sample frames from proposals in the video.
+
+    Required keys are "total_frames" and "out_props", added or modified keys
+    are "frame_inds", "frame_interval", "num_clips", 'clip_len'
+    and 'num_proposals'.
+
+    Args:
+        clip_len (int): Frames of each sampled output clip.
+        body_segments (int): Number of segments in course period.
+        aug_segments (list[int]): Number of segments in starting and
+            ending period.
+        aug_ratio (int | float | Sequence[int | float]): The ratio
```

Comment:
Is it better to define `aug_ratio` as (int | float | tuple[int | float])? since `aug_ratio` may not be changed.